workspace-manager-clienttests/src/main/resources/servers/workspace-local.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "workspace-dev",
+  "description": "Workspace Dev env.",
```

Comment:
This should be consistent with the file name.

Reply:
Updated name and description fields.